src/backend/access/external/pxffilters.c

```diff
@@ -357,6 +393,46 @@ pxf_serialize_filter_list(List *expressionItems)
 
 		switch (tag)
 		{
+			case T_Var:
+			{
+				elog(DEBUG1, "pxf_serialize_filter_list: node tag %d (T_Var)", tag);
+				PxfFilterDesc *filter = (PxfFilterDesc *) palloc0(sizeof(PxfFilterDesc));
+				Var *var = (Var *) node;
+				if (var_to_pxffilter(var, filter))
+				{
+					PxfOperand l = filter->l;
+					PxfOperand r = filter->r;
+					PxfOperatorCode o = filter->op;
+					if (pxfoperand_is_attr(l) && pxfoperand_is_scalar_const(r))
```

Comment:
Left operand constant and right operand is attribute translates in SQL:
WHERE (1,2,3) IN c1 which has incorrect syntax and should be never the case.